squeakroad/docker-compose.yml

```
@@ -0,0 +1,29 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: $APP_SQUEAKROAD_IP
+      APP_PORT: $APP_SQUEAKROAD_PORT
```

We want to bypass Umbrel auth for this app right? Sp people who don't have access to the Umbrel dashboard are still able to access the app? If so we need to add this:

```suggestion
      APP_PORT: $APP_SQUEAKROAD_PORT
      PROXY_AUTH_ADD: "false"
```

---

Yes, that's right.